ui/app/templates/partials/role-pki/form.hbs

```diff
@@ -26,16 +26,5 @@
         Cancel
       {{/secret-link}}
     </div>
-    {{#if (and (eq mode 'edit') model.canDelete)}}
```

Comment:
So no deletes currently happen on the edit page anymore, is that right?

Reply:
Yep, that's right. If there is a "show" page we should have it as a link there instead. We do still have a couple places that still need to be moved I think.